AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -293,6 +282,27 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
 print("Using %s as the reference antenna "%(ant_list[0]))
 h5.select(compscans='interferometric_pointing',ants=ant_list)
 
+if len(opts.channel_mask)>0:
+    pickle_file = open(opts.channel_mask , "rb" )
+    channel_flags = pickle.load(pickle_file)
+    pickle_file.close()
+    NC = len(h5.channels)
+    NF = len(channel_flags)
+    if (NC != NF):
+        print("Warning channel mask (%d) is stretched to fit dataset (%d)!"%(NF,NC))
+        N = NC/float(NF)
+        channel_flags = np.repeat(channel_flags, int(N+0.5)) if (N > 1) else channel_flags[::int(1/N)]
+    rfi_static_flags = channel_flags[:NC] # Clip, just in case
+    for chunk in range(chunks):
+        freq = slice(chunk*(NC//chunks),(chunk+1)*(NC//chunks))
+        masked_f = h5.freqs[freq][rfi_static_flags[freq]]
+        if (len(masked_f) > 0):
+            print("\tFreq. chunk %d: mask omits (%.1f - %.1f)MHz"%(chunk,np.min(masked_f)/1e6,np.max(masked_f)/1e6))
+        else:
+            print("\tFreq. chunk %d: mask omits nothing"%chunk)
+else:
+    rfi_static_flags = np.tile(False, np.shape[1])
```

Comment:
There is a bug (np instead of h5), and full is better than tile for constructing new arrays:
```
rfi_static_flags = np.full(h5.shape[1], False)
```